jenkins/jobs/projects.yaml

```diff
@@ -1,3 +1,17 @@
+- project:
+    name: cinder
+    github-org: openstack
+    zuul-branch:
+      - current
+    node:
+      - ubuntu-xenial
+    vendor: cisco
+
+    jobs:
+        - 'gate-{name}-dsvm-tempest-smoke-{job-name}-{zuul-branch}-{node}':
+            job-name: zm_cinder
+            node: cinder-node
```

Comment:
Currently I've called this "cinder-zonemanager-node" to make it specific its for the zm code, so we'll need to update this line.